scripts/pre_commit_linter.py

```diff
@@ -470,15 +470,17 @@ def _redirect_stdout(new_target):
         sys.stdout = old_target
 
 
-def _get_all_files(input_files, input_path):
-    """This function is used to return the files which needs to linted and
+def _get_all_files(input_path, input_files):
+    """This function is used to return the files which needs to be linted and
     checked.
 
     Args:
-        input_files: list. The list of files to be linted and checked.
         input_path: str. The path of the directory to be linted and checked.
+        input_files: list(str). The list of files to be linted and checked,
```

Comment:
Please replace "files" with "filenames" or "filepaths" throughout (whichever is correct) -- in the function name, the input args and the docstring.

Reply:
(Here and elsewhere. This includes variable names within functions.)

Reply:
Done